images.go

```diff
@@ -203,7 +203,7 @@ var Images = []Image{
 		Tags: []Tag{
 			Tag{
 				Sha: "a325153f1f1e2edb76a76ad789aff172b89dd6178e8f74d39ef87c04d87d1961",
```

Comment:
You were right the first time, we switch both the `sha` and `tag` here usually:
https://github.com/giantswarm/retagger/pull/138/files

Reply:
Fine, I'll revert the revert. This is a bit comfusing